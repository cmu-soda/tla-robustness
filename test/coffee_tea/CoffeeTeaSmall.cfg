SPECIFICATION Spec
INVARIANT TypeOK
INVARIANT WaterSafeTemp
\*INVARIANT OnlyErrorsInv
\*PROPERTY WaterSafeTempProp
