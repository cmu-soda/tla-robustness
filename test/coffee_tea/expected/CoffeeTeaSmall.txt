TLA+ Module: CoffeeTeaSmall
Diff rep grouped by action:

BrewTea:
Safety boundary representation: out/CoffeeTeaSmall_BrewTea_diff_rep.txt
Safety boundary rep formula (with sort definitions):
/\ brewed = {}
/\ temp = 70
